scanner/updater/manual/manual.go

```diff
@@ -0,0 +1,50 @@
+package manual
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/zlog"
+)
+
+// Factory is the UpdaterSetFactory exposed by this package.
+//
+// All configuration is done on the returned updaters.
+type Factory struct {
+}
+
+func UpdaterSet(_ context.Context, vulns []*claircore.Vulnerability) (driver.UpdaterSet, error) {
```

Comment:
What is the purpose of passing in `vulns`? I don't think it's used, is it?

Reply:
It's used in scanner/updater/manual/manual_example_test.go
```
store := postgres.NewMatcherStore(pool)
	appendUpdaterSet(UpdaterSet(ctx, manuallyTestVulns))
```

Reply:
ah ok. I'm not a big fan of designing the API to make it fit the example, but I won't block the PR over this. We can always update it afterwards